lib/licenses.nix

```diff
@@ -55,6 +55,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     free = false;
   };
 
+  angle = {
+    fullName = "ANGLE";
+    url = "https://github.com/google/angle/blob/master/LICENSE";
```

Comment:
This license seems to be `bsd3`, wikipedia also register it as BSD 3 https://en.wikipedia.org/wiki/ANGLE_(software)

Reply:
Thanks @marsam. I changed it to `bsd3`